androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/Extensions.kt

```diff
@@ -58,3 +60,30 @@ fun Context.setNavigationBarVisibility(showNavigationBar: Boolean) {
         }
     }
 }
+fun Context.setActivityBrightness(brightness: Float) {
+    val window = this.findActivity()?.window
+
+    window?.let {
+        val layout: WindowManager.LayoutParams? = it.attributes
+        layout?.screenBrightness = brightness.coerceIn(0f, 1f)
+        it.attributes = layout
+    }
+}
+
+fun Context.resetActivityBrightnessToDefault() {
+    val window = this.findActivity()?.window
+
+    window?.let {
+        val layout: WindowManager.LayoutParams? = it.attributes
+        layout?.screenBrightness = BRIGHTNESS_OVERRIDE_NONE
+        layout?.buttonBrightness = BRIGHTNESS_OVERRIDE_NONE
+        it.attributes = layout
+    }
+}
+
+fun Context.getActivityBrightnessOrDefault(default: Float): Float {
```

Comment:
IMO: it makes more sense to have the default here

```suggestion
fun Context.getActivityBrightnessOrDefault(default: Float = MED_BRIGHTNESS): Float {
```